Gemfile

```diff
@@ -6,9 +6,11 @@ gem 'rails', '~> 4.2.6'
 gem 'activerecord-session_store'
 gem 'ahoy_matey'
 gem 'american_date'
+gem 'bcrypt'
 gem 'browserify-rails'
 gem 'coffee-rails', '~> 4.1.0'
 gem 'devise', '~> 4.1'
+gem 'devise-encryptable', github: '18F/devise-encryptable', branch: 'pbkdf2-encryptor'
```

Comment:
Was going to ask if this is something we can PR back to the main lib, but I see you already did that! https://github.com/plataformatec/devise-encryptable/pull/12 

Maybe add Pr link to Gemfile in case we forget to switch this out when it is merged?